### ISSUE TITLE
I want to mask the map on the page to have the precise outline of a map to be able to stitch different maps together

### ISSUE THREAD

**Issue by glenrobson** [IIIF/iiif-stories#127]
# Description

I want to mask the map on the page to have the precise outline of a map to be able to stitch different maps together. 
 * Clean off ragged edges  
 * Allow non rectangle areas  
 * We want to use the annotation functionality for masking.  
 * Mask on original image (pre-georeferencing) -- to discuss 

12-votes in Stanford maps meeting

## Variation(s)

(do you know of, or can you imagine, similar use cases?)

## Proposed Solutions

(any ideas about how your use case might be supported)

## Additional Background

(more about your perspective, existing work, etc. goes here.)

**Comment by glenrobson** [comment]
Still to determine motivation but should indicate this is a masking annotation. No body yet but would have a body if it was geo-referenced

```
{
   "id": "http://annotation/id", 
   "motivation": "masking",
   "target": {
    "source": "http://example.org/map1",
    "selector": {
      "type": "SvgSelector",
      "value": "<svg:svg> ... </svg:svg>"
    }
  }
}
```

**Comment by mejackreed** [comment]
Much of this is handled for georeferenced content using approach in #129 .

**Comment by mprove** [comment]
Yes and no. The user interface should handle both cases, alas displaying a map with and without the "white" margins, as important info might be chopped off otherwise.
Furthermore a set of map sheets needs to be an entity for the viewer. Each sheet might have its own Permalink and metadata. All together they might be contained in the same IIIF manifest (e.g. an atlas).
